qtodotxt2/main_controller.py

```diff
@@ -325,3 +326,10 @@ def completeTasks(self, tasks):
             else:
                 task.setPending()
 
+    docPathChanged = QtCore.pyqtSignal()
+
+    @QtCore.pyqtProperty('QUrl', notify=docPathChanged)
```

Comment:
It's the folder location which is opened in the open/saveas dialog. (depending on your DE e.g. ~/Documents). 
My idea was, that you could set it to the location of the current/last todo.txt
If you don't want that, we can set the notify=None